src/wallet/wallet.h

```diff
@@ -35,22 +35,13 @@
 typedef CWallet* CWalletRef;
 extern std::vector<CWalletRef> vpwallets;
 
-/**
- * Settings
- */
-extern CFeeRate payTxFee;
-extern unsigned int nTxConfirmTarget;
-extern bool bSpendZeroConfChange;
-extern bool fWalletRbf;
-extern bool g_wallet_allow_fallback_fee;
-
 //! Default for -keypool
 static const unsigned int DEFAULT_KEYPOOL_SIZE = 1000;
 //! -paytxfee default
-static const CAmount DEFAULT_TRANSACTION_FEE = 0;
+constexpr CAmount DEFAULT_PAY_TX_FEE = 0;
```

Comment:
Seems a little weird to change this to `constexpr` and leave all the other default constants as `static const`. It's fine, but can we have a follow-up PR to change the remaining constants to `constexpr`s?

Reply:
Sure. There is a bit of refactoring going on in the wallet, so I will wait until the dust settled a bit.